syntax-highlighting-code-block.php

```diff
@@ -678,9 +672,6 @@ function render_block( array $attributes, string $content ): string {
 	 * @param string[] $auto_detect_language Auto-detect languages.
 	 */
 	$auto_detect_languages = apply_filters( 'syntax_highlighting_code_block_auto_detect_languages', [] );
-	if ( ! is_array( $auto_detect_languages ) ) {
-		$auto_detect_languages = [];
-	}
```

Comment:
With these lines present, PHPStan complains with:

```
Call to function is_array() with array<string> will always evaluate to true.                                                                     
💡 Because the type is coming from a PHPDoc, you can turn off this check by setting treatPhpDocTypesAsCertain: false in your phpstan.neon.dist.   
```

However, it's entirely possible that a plugin could do something unexpected, for example:

```php
add_filter( 'syntax_highlighting_code_block_auto_detect_languages', '__return_false' );
```

If so, then a fatal error would ensue:

```
Fatal error: Uncaught TypeError: array_filter(): Argument #1 ($array) must be of type array, bool given
```

Therefore, for the most strict and safe possible code, it seems preferable to keep this.